pep-0683.rst

```diff
@@ -72,14 +72,19 @@ Here's a high-level look at the implementation:
 If an object's refcount matches a very specific value (defined below)
 then that object is treated as immortal.  The CPython C-API and runtime
 will not modify the refcount (or other runtime state) of an immortal
-object.
+object.  The runtime will now be explicitly responsible for deallocating
+all immortal objects during finalization, unless statically allocated.
+(See `Object Cleanup`_ below.)
 
 Aside from the change to refcounting semantics, there is one other
 possible negative impact to consider.  The threshold for an "acceptable"
 performance penalty for immortal objects is 2% (the consensus at the
 2022 Language Summit).  A naive implementation of the approach described
-below makes CPython roughly 6% slower.  However, the implementation
-is performance-neutral once known mitigations are applied.
+below makes CPython roughly 4% slower.  However, the implementation
```

Comment:
This is probably much much worse today, maybe we should re-measure? Or just remove the 4% completely and mention that it's "significantly slower"?

Reply:
I think I'll leave "roughly 4%".  An exactly accurate value doesn't communicate anything extra and "significantly slower" doesn't offer any good scale.